Tests/SWBBuildSystemTests/SwiftDriverTests.swift

```diff
@@ -4610,7 +4608,7 @@ fileprivate struct SwiftDriverTests: CoreBasedTests {
 
             for warningLevel in [BooleanWarningLevel.yes, .yesError] {
                 let parameters = BuildParameters(configuration: "Debug", overrides: ["DIAGNOSE_MISSING_TARGET_DEPENDENCIES": warningLevel.rawValue])
-                let buildRequest = BuildRequest(parameters: parameters, buildTargets: tester.workspace.projects[0].targets.map({ BuildRequest.BuildTargetInfo(parameters: parameters, target: $0) }), continueBuildingAfterErrors: false, useParallelTargets: true, useImplicitDependencies: false, useDryRun: false)
+                let buildRequest = BuildRequest(parameters: parameters, buildTargets: tester.workspace.projects[0].targets.map({ BuildRequest.BuildTargetInfo(parameters: parameters, target: $0) }), continueBuildingAfterErrors: false, useParallelTargets: false, useImplicitDependencies: false, useDryRun: false)
```

Comment:
Relying on deprecated manual ordering isn't ideal, but it's better than script phases calling sleep IMO